### PR TITLE
Add UI to perform software checks

### PR DESCRIPTION
**Pull request by Ishadijcks**
## Introduction

Add an rqt plugin to the `march_launch` package.

Run with
```
roslaunch march_launch march_rqt_software_check.launch 
```

## Features
- Run software checks from a separate thread.
- Start launch files within a check
- Manual confirmation if a check needs it.
- Present information to the screen from within the check.
- Inheritance to make different types of checks easier.
- Colors that indicate which checks are already performed.

### CODE REVIEW

**Comment by TimBuckers** [comment]
We can add a tutorial on how to create a new software test here: https://github.com/project-march/tutorials

File: march_launch/src/march_launch/checks/SlaveCountCheck.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            slave_count = rospy.get_param("/check/slave_count")
+        except KeyError:
+            self.log("Could not find key /check/slave_count", Color.Error)
+            self.passed = False
+            self.done = True
+            return
+
+        self.log("Ethercat found " + str(slave_count) + " slave(s)", Color.Info)
+        self.passed = slave_count > 0
+        self.done = True
```
Comment (JitskedeVries) [inline march_launch/src/march_launch/checks/SlaveCountCheck.py]: Have you tested this on the exoskeleton?

**Comment by martijnvandermarel** [comment]
Can we get a demo of this on the exoskeleton before merging it?

**Comment by Ishadijcks** [comment]
Of course, feel free to come down here

File: march_launch/src/march_launch/checks/SlaveCountCheck.py (hunk 1 above)
Comment (Ishadijcks) [reply march_launch/src/march_launch/checks/SlaveCountCheck.py]: Yes

Comment (Ishadijcks) [reply march_launch/src/march_launch/checks/SlaveCountCheck.py]: See the above gif

**Pull request merged**